src/index.html

```diff
@@ -1,13 +1,140 @@
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__list-tag" >
+              gadgets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__list-tag" >
+              tablets
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__list-tag" >
+              photo
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a href="#" class="nav__list-tag" >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="card-container">
+
+      <div class="card" data-qa="card">
+        <p class="card__image-box">
+          <img src="./images/imac.jpeg"
+          alt="Mac"
+          class="card__image-box-mac-image"
+          >
+        </p>
+        <article class="card__info">
+          <p class="card__mac-model">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+          <div class="stars">
+            <ul class="stars__list">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <p class="stars__text">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-total">$2,199</p>
+          </div>
+          <div class="button">
+            <a class="button__buy" data-qa="hover">buy</a>
+          </div>
+        </article>
+      </div>
+
+      <div class="card" data-qa="card">
+        <p class="card__image-box">
+          <img src="./images/imac.jpeg"
+          alt="Mac"
+          class="card__image-box-mac-image"
+          >
+        </p>
+        <article class="card__info">
+          <p class="card__mac-model">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+          <div class="stars">
+            <ul class="stars__list">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+            <p class="stars__text">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-total">$2,199</p>
+          </div>
+          <div class="button">
+            <a class="button__buy" data-qa="hover">buy</a>
+          </div>
```

Comment:
it's not a button, it is a link 
for `a` tag there should be `href`  

```suggestion
           <a
        class="button__buy"
        href="#"
        data-qa="hover"
      >
```